### PR TITLE
PGAND-259 iOS Client Text Updates

### PR DESCRIPTION
**Pull request by CelesteTang**
# Story
https://jira.mozilla.com/browse/PGAND-259

# Implementation
1. update app display name
2. update Localizable.strings files
3. update README

### CODE REVIEW

File: FirefoxPrivateNetworkVPN/ViewControllers/UpdateRequiredViewController.xib [hunk 1]
```diff
@@ -81,15 +81,15 @@
...
+                    <rect key="frame" x="179.5" y="625.5" width="665" height="21.5"/>
                     <constraints>
                         <constraint firstAttribute="width" relation="lessThanOrEqual" constant="665" id="lcf-GJ-ZZh"/>
                     </constraints>
                     <attributedString key="attributedText">
                         <fragment content="Please update the app before you continue to use the VPN">
                             <attributes>
                                 <color key="NSColor" name="custom_grey40"/>
-                                <font key="NSFont" size="15" name="Inter-Regular"/>
+                                <font key="NSFont" metaFont="system" size="15"/>
```
Comment (Conjuror) [inline FirefoxPrivateNetworkVPN/ViewControllers/UpdateRequiredViewController.xib:92]: Why do we change font here?

File: FirefoxPrivateNetworkVPN/ViewControllers/UpdateRequiredViewController.xib [hunk 2]
```diff
@@ -115,7 +115,7 @@
                                 <fragment content="Your connection will not be secure while you update.">
                                     <attributes>
                                         <color key="NSColor" name="custom_grey50"/>
-                                        <font key="NSFont" size="13" name="Inter-Regular"/>
+                                        <font key="NSFont" metaFont="system"/>
```
Comment (Conjuror) [inline FirefoxPrivateNetworkVPN/ViewControllers/UpdateRequiredViewController.xib:118]: Why do we change font here?

File: FirefoxPrivateNetworkVPN/ViewControllers/UpdateRequiredViewController.xib (hunk 1 above)
Comment (Conjuror) [inline FirefoxPrivateNetworkVPN/ViewControllers/UpdateRequiredViewController.xib:92]: Why do we change font here?

File: FirefoxPrivateNetworkVPN/ViewControllers/UpdateRequiredViewController.xib [hunk 3]
```diff
@@ -281,6 +281,9 @@
         <namedColor name="custom_grey50">
             <color red="0.047058823529411764" green="0.047058823529411764" blue="0.050980392156862744" alpha="0.80000001192092896" colorSpace="custom" customColorSpace="sRGB"/>
         </namedColor>
+        <namedColor name="custom_grey50">
```
Comment (Conjuror) [inline FirefoxPrivateNetworkVPN/ViewControllers/UpdateRequiredViewController.xib:284]: Seems like a duplicated color definition?

File: FirefoxPrivateNetworkVPN/ViewControllers/UpdateRequiredViewController.xib (hunk 2 above)
Comment (CelesteTang) [reply FirefoxPrivateNetworkVPN/ViewControllers/UpdateRequiredViewController.xib]: I've removed unnecessary paragraph style and set up correct fonts.

File: FirefoxPrivateNetworkVPN/ViewControllers/UpdateRequiredViewController.xib (hunk 1 above)
Comment (CelesteTang) [reply FirefoxPrivateNetworkVPN/ViewControllers/UpdateRequiredViewController.xib]: I've removed unnecessary paragraph style and set up correct fonts.

File: FirefoxPrivateNetworkVPN/ViewControllers/UpdateRequiredViewController.xib (hunk 3 above)
Comment (CelesteTang) [reply FirefoxPrivateNetworkVPN/ViewControllers/UpdateRequiredViewController.xib]: I've removed the duplicated color definition. Thank you!

**Pull request closed without merging**